internal/status-tracker/Cargo.toml

```diff
@@ -21,3 +21,4 @@ all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
 
 [dependencies]
+c2pa-crypto = { path = "../crypto", version = "0.1.2" }
```

Comment:
As you mentioned separately, this creates a circular dependency that won't work.